src/SerilogWeb.Classic/SerilogWeb.Classic.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>SerilogWeb.Classic</id>
-    <version>$version$</version>
+    <version>2.0.3-private2</version>
```

Comment:
Builds run through _AppVeyor_ so this is all done automatically. Needs a revert here as package versioning won't work for us with this change. To pack this spec with the `$version$` token in there, you can pass the `-Version` parameter to `nuget.exe pack` IIRC.